src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const pagesList = [];
+
+  for (let i = 1; i <= totalPages; i++) {
+    pagesList.push(i);
+  }
```

Comment:
Consider using `Array.from` to create the `pagesList` array more concisely. For example: `const pagesList = Array.from({ length: totalPages }, (_, index) => index + 1);`